specification/widget/resource-manager/Microsoft.Widget/WidgetManagement/main.tsp

```diff
@@ -0,0 +1,35 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "./employee.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+
+/** Microsoft.Contoso Resource Provider management API. */
+@armProviderNamespace
+@service(#{ title: "WidgetManagement" })
+@versioned(WidgetManagement.Versions)
+namespace WidgetManagement;
```

Comment:
despite this is a demo, we should ensure the typespec is correct.
Here we should either change the namespace here to its provider namespace (`Microsoft.Widget`, the same as we have in the filepath), or add an argument to the `armProviderNamespace` decorator

Reply:
I agree we want this sample to be as correct as possible, including the TSP itself, and tspconfig.yaml.  However, this raises the point, we may need to validate the TSP namespace matches the service name and/or rp namespace.